CryptoTechReminderSystem.Test/HarvestGatewayTests.cs

```diff
@@ -32,6 +35,17 @@ private void SetUpUsersApiEndpoint(string id, string firstName, string lastName,
             var json = $"{{  \"users\":[    {{      \"id\":{id},      \"first_name\":\"{firstName}\",      \"last_name\":\"{lastName}\",      \"email\":\"{email}\"    }}  ]}}";
```

Comment:
This line is messy. Perhaps splitting this long line onto multiple lines is a good idea(?).

Reply:
Good point, all done! d0f4bca9ae297a62794967796aa35e98410f2154